protos/address/address_helpers.go

```diff
@@ -0,0 +1,21 @@
+package address
+
+import (
+	"fmt"
+)
+
+// String returns a human-readable representation of the Address.
+func (a *Address) Strings() string {
```

Comment:
Typo? Comment says String, function says Strings. Typically the interface to implement is "String".
```suggestion
func (a *Address) String() string {
```